plugins/storage/volume/solidfire/src/org/apache/cloudstack/storage/datastore/driver/SolidFirePrimaryDataStoreDriver.java

```diff
@@ -352,8 +352,8 @@ public long getVolumeSizeIncludingHypervisorSnapshotReserve(Volume volume, Stora
         Integer hypervisorSnapshotReserve = volume.getHypervisorSnapshotReserve();
 
         if (hypervisorSnapshotReserve != null) {
-            if (hypervisorSnapshotReserve < 50) {
-                hypervisorSnapshotReserve = 50;
+            if (hypervisorSnapshotReserve < 10) {
+                hypervisorSnapshotReserve = 10;
```

Comment:
Why not use Math.max(hypervisorSnapshotReserve, 10)? That way we will be using the const only once and its more readable( to me ;) )